File: Windows phone/442/442/442/Api/Models/Live.cs
using System.Collections.Generic;

namespace _442.Api.Models
{
    public class Live
    {
        #region Properties
        public string HomeTeamName { get; set; }
        public string AwayTeamName { get; set; }
        public int HomeTeamScore { get; set; }
        public int AwayTeamScore { get; set; }
        public long MatchId { get; set; }
        public string MatchStatus { get; set; }
        public List<string> HomeTeamGoalScorers { get; set; }
        public List<string> AwayTeamGoalScorers { get; set; }
        public List<Comment> Commentry { get; set; }
        #endregion Properties

        #region Constructor

        public Live(string home, string away, int homeScore, int awayScore, long matchId)
        {
            HomeTeamName = home;
            AwayTeamName = away;
            HomeTeamScore = homeScore;
            AwayTeamScore = awayScore;
            MatchId = matchId;
        }

        #endregion Constructor
    }
}

File: Android/Soccer_442/app/src/main/java/com/nyoka/soccer_442/Live.java
package com.nyoka.soccer_442;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/**
 * Created by linda.nyoka on 2015-02-22.
 */
public class Live implements Serializable {

    public String HomeTeamName;
    public String AwayTeamName ;
    public int HomeTeamScore;
    public int AwayTeamScore;
    public long MatchId;
    public String MatchStatus;
    public List<String> HomeTeamGoalScorers ;
    public List<String> AwayTeamGoalScorers;
    public ArrayList<Comment> Commentry;


    public Live(String home, String away, int homeScore, int awayScore, long matchId)
    {
        HomeTeamName = home;
        AwayTeamName = away;
        HomeTeamScore = homeScore;
        AwayTeamScore = awayScore;
        MatchId = matchId;
    }
}
